### ISSUE TITLE
Install Issue on Windows with Strawberry

### ISSUE THREAD

**Issue by genio** [Perl5-Alien/Alien-Build#181]
I'm using PSPerl (a perlbrew-like app to switch between differing versions of Strawberry Portable installs).

In attempt to install Alien::Build, I'm running into a problem with long file paths causing errors during testing of Alien::Build. Granted, these paths are _very_ long and this might not happen for others, so YMMV:

```log
PS C:\Users\cwhitener\chase\Alien-libuv> cpanm -v Alien::Build
cpanm (App::cpanminus) 1.7044 on perl 5.030000 built for MSWin32-x86-multi-thread-64int
Work directory is C:\Users\cwhitener\_psperl\_perls\perl32w64int-5.30.0.1\data/.cpanm/work/1584319397.3836
You have make C:\Users\cwhitener\_psperl\_perls\perl32w64int-5.30.0.1\c\bin\gmake.exe
You have LWP 6.39
Falling back to Archive::Tar 2.32
Searching Alien::Build () on cpanmetadb ...
--> Working on Alien::Build
Fetching http://www.cpan.org/authors/id/P/PL/PLICEASE/Alien-Build-2.14.tar.gz ... OK
Unpacking Alien-Build-2.14.tar.gz
Entering Alien-Build-2.14
Checking configure dependencies from META.json
Checking if you have ExtUtils::CBuilder 0 ... Yes (0.280231)
Checking if you have ExtUtils::MakeMaker 6.64 ... Yes (7.36)
Checking if you have File::Which 0 ... Yes (1.23)
Checking if you have ExtUtils::ParseXS 3.30 ... Yes (3.40)
Running Makefile.PL
Configuring Alien-Build-2.14 ... Checking if your kit is complete...
Looks good
Generating a gmake-style Makefile
Writing Makefile for Alien::Build
Writing MYMETA.yml and MYMETA.json
OK
Checking dependencies from MYMETA.json ...
Checking if you have FFI::CheckLib 0.11 ... Yes (0.24)
Checking if you have ExtUtils::ParseXS 3.30 ... Yes (3.40)
Checking if you have List::Util 1.33 ... Yes (1.50)
Checking if you have Test2::V0 0.000060 ... Yes (0.000122)
Checking if you have ExtUtils::MakeMaker 6.64 ... Yes (7.36)
Checking if you have Test2::API 1.302096 ... Yes (1.302164)
Checking if you have File::Which 1.10 ... Yes (1.23)
Checking if you have Capture::Tiny 0.17 ... Yes (0.48)
Checking if you have Path::Tiny 0.077 ... Yes (0.108)
Checking if you have PkgConfig 0.14026 ... Yes (0.23026)
Checking if you have ExtUtils::CBuilder 0 ... Yes (0.280231)
Checking if you have File::chdir 0 ... Yes (0.1010)
Checking if you have JSON::PP 0 ... Yes (4.02)
Checking if you have Text::ParseWords 3.26 ... Yes (3.30)
Building and testing Alien-Build-2.14 ... cp lib/Alien/Build/Manual/Alien.pod blib\lib/Alien/Build/Manual/Alien.pod
cp lib/Alien/Build/Manual/Contributing.pod blib\lib/Alien/Build/Manual/Contributing.pod
cp lib/Alien/Build/Log/Default.pm blib\lib/Alien/Build/Log/Default.pm
cp lib/Alien/Build/Interpolate.pm blib\lib/Alien/Build/Interpolate.pm
cp lib/Alien/Base/FAQ.pod blib\lib/Alien/Base/FAQ.pod
cp lib/Alien/Base/PkgConfig.pm blib\lib/Alien/Base/PkgConfig.pm
cp lib/Alien/Base/Wrapper.pm blib\lib/Alien/Base/Wrapper.pm
cp lib/Alien/Build/Log.pm blib\lib/Alien/Build/Log.pm
cp lib/Alien/Base.pm blib\lib/Alien/Base.pm
cp lib/Alien/Build.pm blib\lib/Alien/Build.pm
cp lib/Alien/Build/Manual/AlienAuthor.pod blib\lib/Alien/Build/Manual/AlienAuthor.pod
cp lib/Alien/Build/CommandSequence.pm blib\lib/Alien/Build/CommandSequence.pm
cp lib/Alien/Build/Manual/AlienUser.pod blib\lib/Alien/Build/Manual/AlienUser.pod
cp lib/Alien/Build/Manual/FAQ.pod blib\lib/Alien/Build/Manual/FAQ.pod
cp lib/Alien/Build/Log/Abbreviate.pm blib\lib/Alien/Build/Log/Abbreviate.pm
cp lib/Alien/Build/Interpolate/Default.pm blib\lib/Alien/Build/Interpolate/Default.pm
cp lib/Alien/Base/Authoring.pod blib\lib/Alien/Base/Authoring.pod
cp lib/Alien/Build/MM.pm blib\lib/Alien/Build/MM.pm
cp lib/Alien/Build/Plugin/Core/FFI.pm blib\lib/Alien/Build/Plugin/Core/FFI.pm
cp lib/Alien/Build/Plugin/Build/Autoconf.pm blib\lib/Alien/Build/Plugin/Build/Autoconf.pm
cp lib/Alien/Build/Plugin/Core/Override.pm blib\lib/Alien/Build/Plugin/Core/Override.pm
cp lib/Alien/Build/Plugin/Build/Make.pm blib\lib/Alien/Build/Plugin/Build/Make.pm
cp lib/Alien/Build/Plugin.pm blib\lib/Alien/Build/Plugin.pm
cp lib/Alien/Build/Plugin/Core/Legacy.pm blib\lib/Alien/Build/Plugin/Core/Legacy.pm
cp lib/Alien/Build/Plugin/Build/CMake.pm blib\lib/Alien/Build/Plugin/Build/CMake.pm
cp lib/Alien/Build/Plugin/Core/Download.pm blib\lib/Alien/Build/Plugin/Core/Download.pm
cp lib/Alien/Build/Plugin/Build.pod blib\lib/Alien/Build/Plugin/Build.pod
cp lib/Alien/Build/Plugin/Build/MSYS.pm blib\lib/Alien/Build/Plugin/Build/MSYS.pm
cp lib/Alien/Build/Manual/PluginAuthor.pod blib\lib/Alien/Build/Manual/PluginAuthor.pod
cp lib/Alien/Build/Plugin/Core/CleanInstall.pm blib\lib/Alien/Build/Plugin/Core/CleanInstall.pm
cp lib/Alien/Build/Plugin/Core/Gather.pm blib\lib/Alien/Build/Plugin/Core/Gather.pm
cp lib/Alien/Build/Plugin/Core.pod blib\lib/Alien/Build/Plugin/Core.pod
cp lib/Alien/Build/Plugin/Build/SearchDep.pm blib\lib/Alien/Build/Plugin/Build/SearchDep.pm
cp lib/Alien/Build/Plugin/Decode.pod blib\lib/Alien/Build/Plugin/Decode.pod
cp lib/Alien/Build/Plugin/Decode/DirListing.pm blib\lib/Alien/Build/Plugin/Decode/DirListing.pm
cp lib/Alien/Build/Plugin/Extract/ArchiveZip.pm blib\lib/Alien/Build/Plugin/Extract/ArchiveZip.pm
cp lib/Alien/Build/Plugin/Extract/CommandLine.pm blib\lib/Alien/Build/Plugin/Extract/CommandLine.pm
cp lib/Alien/Build/Plugin/Decode/HTML.pm blib\lib/Alien/Build/Plugin/Decode/HTML.pm
cp lib/Alien/Build/Plugin/Core/Setup.pm blib\lib/Alien/Build/Plugin/Core/Setup.pm
cp lib/Alien/Build/Plugin/Download/Negotiate.pm blib\lib/Alien/Build/Plugin/Download/Negotiate.pm
cp lib/Alien/Build/Plugin/Extract.pod blib\lib/Alien/Build/Plugin/Extract.pod
cp lib/Alien/Build/Plugin/Decode/DirListingFtpcopy.pm blib\lib/Alien/Build/Plugin/Decode/DirListingFtpcopy.pm
cp lib/Alien/Build/Plugin/Download.pod blib\lib/Alien/Build/Plugin/Download.pod
cp lib/Alien/Build/Plugin/Decode/Mojo.pm blib\lib/Alien/Build/Plugin/Decode/Mojo.pm
cp lib/Alien/Build/Plugin/Core/Tail.pm blib\lib/Alien/Build/Plugin/Core/Tail.pm
cp lib/Alien/Build/Plugin/Extract/ArchiveTar.pm blib\lib/Alien/Build/Plugin/Extract/ArchiveTar.pm
cp lib/Alien/Build/Plugin/Extract/Directory.pm blib\lib/Alien/Build/Plugin/Extract/Directory.pm
cp lib/Alien/Build/Plugin/Fetch/Wget.pm blib\lib/Alien/Build/Plugin/Fetch/Wget.pm
cp lib/Alien/Build/Plugin/Fetch.pod blib\lib/Alien/Build/Plugin/Fetch.pod
cp lib/Alien/Build/Plugin/Fetch/NetFTP.pm blib\lib/Alien/Build/Plugin/Fetch/NetFTP.pm
cp lib/Alien/Build/Plugin/Fetch/LWP.pm blib\lib/Alien/Build/Plugin/Fetch/LWP.pm
cp lib/Alien/Build/Plugin/PkgConfig/MakeStatic.pm blib\lib/Alien/Build/Plugin/PkgConfig/MakeStatic.pm
cp lib/Alien/Build/Plugin/PkgConfig/LibPkgConf.pm blib\lib/Alien/Build/Plugin/PkgConfig/LibPkgConf.pm
cp lib/Alien/Build/Plugin/Fetch/HTTPTiny.pm blib\lib/Alien/Build/Plugin/Fetch/HTTPTiny.pm
cp lib/Alien/Build/Plugin/Gather/IsolateDynamic.pm blib\lib/Alien/Build/Plugin/Gather/IsolateDynamic.pm
cp lib/Alien/Build/Plugin/Fetch/Local.pm blib\lib/Alien/Build/Plugin/Fetch/Local.pm
cp lib/Alien/Build/Plugin/PkgConfig/CommandLine.pm blib\lib/Alien/Build/Plugin/PkgConfig/CommandLine.pm
cp lib/Alien/Build/Plugin/Fetch/CurlCommand.pm blib\lib/Alien/Build/Plugin/Fetch/CurlCommand.pm
cp lib/Alien/Build/Plugin/Fetch/LocalDir.pm blib\lib/Alien/Build/Plugin/Fetch/LocalDir.pm
cp lib/Alien/Build/Plugin/Extract/Negotiate.pm blib\lib/Alien/Build/Plugin/Extract/Negotiate.pm
cp lib/Alien/Build/rc.pm blib\lib/Alien/Build/rc.pm
cp lib/Alien/Build/Plugin/PkgConfig/PP.pm blib\lib/Alien/Build/Plugin/PkgConfig/PP.pm
cp lib/Alien/Build/Util.pm blib\lib/Alien/Build/Util.pm
cp lib/Alien/Build/Plugin/Prefer/BadVersion.pm blib\lib/Alien/Build/Plugin/Prefer/BadVersion.pm
cp lib/Alien/Build/Plugin/Prefer/SortVersions.pm blib\lib/Alien/Build/Plugin/Prefer/SortVersions.pm
cp lib/Alien/Build/Plugin/Probe/Vcpkg.pm blib\lib/Alien/Build/Plugin/Probe/Vcpkg.pm
cp lib/Alien/Build/Plugin/Probe/CBuilder.pm blib\lib/Alien/Build/Plugin/Probe/CBuilder.pm
cp lib/Alien/Build/Plugin/Prefer/GoodVersion.pm blib\lib/Alien/Build/Plugin/Prefer/GoodVersion.pm
cp lib/Alien/Build/Plugin/Prefer.pod blib\lib/Alien/Build/Plugin/Prefer.pod
cp lib/Alien/Build/Plugin/Probe.pod blib\lib/Alien/Build/Plugin/Probe.pod
cp lib/Alien/Build/Plugin/Probe/CommandLine.pm blib\lib/Alien/Build/Plugin/Probe/CommandLine.pm
cp lib/Alien/Build/Version/Basic.pm blib\lib/Alien/Build/Version/Basic.pm
cp lib/Alien/Build/Plugin/Test/Mock.pm blib\lib/Alien/Build/Plugin/Test/Mock.pm
cp lib/Alien/Build/Temp.pm blib\lib/Alien/Build/Temp.pm
cp lib/Alien/Build/Plugin/PkgConfig/Negotiate.pm blib\lib/Alien/Build/Plugin/PkgConfig/Negotiate.pm
cp lib/Test/Alien/CanCompile.pm blib\lib/Test/Alien/CanCompile.pm
cp lib/Test/Alien/Diag.pm blib\lib/Test/Alien/Diag.pm
cp lib/Test/Alien/Synthetic.pm blib\lib/Test/Alien/Synthetic.pm
cp lib/Test/Alien/Build.pm blib\lib/Test/Alien/Build.pm
cp lib/Alien/Role.pm blib\lib/Alien/Role.pm
cp lib/Test/Alien.pm blib\lib/Test/Alien.pm
cp lib/Test/Alien/CanPlatypus.pm blib\lib/Test/Alien/CanPlatypus.pm
cp lib/Test/Alien/Run.pm blib\lib/Test/Alien/Run.pm
cp lib/alienfile.pm blib\lib/alienfile.pm
"C:\Users\cwhitener\_psperl\_perls\perl32w64int-5.30.0.1\perl\bin\perl.exe" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib\lib', 'blib\arch')" t/*.t
t/00_diag.t .......................................... #
t/00_diag.t .......................................... 1/? #
#
# HARNESS_ACTIVE=1
# HARNESS_VERSION=3.42
# PERL5LIB=C:\Users\cwhitener\_psperl\_perls\perl32w64int-5.30.0.1\data\.cpanm\work\1584319397.3836\Alien-Build-2.14\blib\lib;C:\Users\cwhitener\_psperl\_perls\perl32w64int-5.30.0.1\data\.cpanm\work\1584319397.3836\Alien-Build-2.14\blib\arch
# PERL_MM_USE_DEFAULT=1
# PERL_USE_UNSAFE_INC=1
# PSPERL_ROOT=C:\Users\cwhitener\_psperl
#
#
#
# PERL5LIB path
# C:\Users\cwhitener\_psperl\_perls\perl32w64int-5.30.0.1\data\.cpanm\work\1584319397.3836\Alien-Build-2.14\blib\lib
# C:\Users\cwhitener\_psperl\_perls\perl32w64int-5.30.0.1\data\.cpanm\work\1584319397.3836\Alien-Build-2.14\blib\arch
#
#
#
# perl                     5.030000
# Acme::Alien::DontPanic   -
# Alien::Base::ModuleBuild -
# Alien::Libbz2            -
# Alien::cmake3            0.04
# Alien::gzip              -
# Alien::xz                -
# Archive::Tar             2.32
# Archive::Zip             1.64
# Capture::Tiny            0.48
# Devel::Hide              -
# Env::ShellWords          -
# ExtUtils::CBuilder       0.280231
# ExtUtils::MakeMaker      7.36
# ExtUtils::ParseXS        3.40
# FFI::CheckLib            0.24
# FFI::Platypus            -
# File::Listing            6.04
# File::Listing::Ftpcopy   -
# File::Which              1.23
# File::chdir              0.1010
# HTML::Parser             3.72
# HTTP::Tiny               0.076
# IO::Compress::Bzip2      2.086
# IO::Socket::SSL          2.066
# IO::Uncompress::Bunzip2  2.086
# IO::Zlib                 1.10
# JSON::PP                 4.02
# LWP                      6.39
# LWP::Protocol::https     6.07
# List::Util               1.5
# Mojo::DOM58              -
# Mojolicious              8.16
# Net::FTP                 3.11
# Net::SSLeay              1.88
# Path::Tiny               0.108
# PkgConfig                0.23026
# PkgConfig::LibPkgConf    -
# Readonly                 2.05
# Sort::Versions           1.62
# Test2::API               1.302164
# Test2::V0                0.000122
# Text::ParseWords         3.30
# URI                      1.76
# YAML                     1.29
#
#
#
# ---
# cmake_generator: MinGW Makefiles
# compiler_type: unix
# pkg-config:
#   pkg-config: C:\Users\cwhitener\_psperl\_perls\perl32w64int-5.30.0.1\perl\bin\pkg-config.BAT
#   pkgconf: ~
# system_type: windows-strawberry
# pkg-config negotiate pick = PkgConfig::PP
#
#
# [config.site]
# # file automatically generated by C:\Users\cwhitener\_psperl\_perls\perl32w64int-5.30.0.1\data\.cpanm\work\1584319397.3836\Alien-Build-2.14\blib\lib/Alien/Build/Plugin/Build/Autoconf.pm
# libdir='${prefix}/lib'
#
#
#
t/00_diag.t .......................................... ok
t/01_use.t ........................................... ok
t/alien_base.t ....................................... ok
t/alien_base__system_installed.t ..................... skipped: test requires Alien::Base::ModuleBuild
t/alien_base_pkgconfig.t ............................. ok
t/alien_base_wrapper.t ............................... ok
t/alien_build.t ...................................... ok
t/alien_build_commandsequence.t ...................... ok
t/alien_build_commandsequence__cd.t .................. ok
t/alien_build_interpolate.t .......................... ok
t/alien_build_interpolate_default.t .................. ok
t/alien_build_log.t .................................. ok
t/alien_build_log_abbreviate.t ....................... ok
t/alien_build_log_default.t .......................... ok
t/alien_build_meta.t ................................. ok
t/alien_build_mm.t ................................... ok
t/alien_build_plugin.t ............................... ok
t/alien_build_plugin_build_autoconf.t ................ 1/?     # Failed test 'alien builds okay'
    # at t/alien_build_plugin_build_autoconf.t line 88.
    # Alien::Build::Plugin::Core::Download> downloaded foo-1.00.tar
    # Alien::Build::Auto::Jwlwv2::Alienfile> ---
    # Alien::Build::Auto::Jwlwv2::Alienfile> autoconf_prefix: /C/Users/cwhitener/_psperl/_perls/perl32w64int-5.30.0.1/data/.cpanm/work/1584319397.3836/Alien-Build-2.14/.tmp/ZjwQGlwvS0/prefix
    # Alien::Build::Auto::Jwlwv2::Alienfile> complete:
    # Alien::Build::Auto::Jwlwv2::Alienfile>   download: 1
    # Alien::Build::Auto::Jwlwv2::Alienfile> download: C:/Users/cwhitener/_psperl/_perls/perl32w64int-5.30.0.1/data/.cpanm/work/1584319397.3836/Alien-Build-2.14/corpus/dist/foo-1.00.tar
    # Alien::Build::Auto::Jwlwv2::Alienfile> extract: C:/Users/cwhitener/_psperl/_perls/perl32w64int-5.30.0.1/data/.cpanm/work/1584319397.3836/Alien-Build-2.14/.tmp/ZjwQGlwvS0/root/extract_eDyO/foo-1.00
    # Alien::Build::Auto::Jwlwv2::Alienfile> patch: ~
    # Alien::Build::Auto::Jwlwv2::Alienfile> prefix: C:/Users/cwhitener/_psperl/_perls/perl32w64int-5.30.0.1/data/.cpanm/work/1584319397.3836/Alien-Build-2.14/.tmp/ZjwQGlwvS0/prefix
    # Alien::Build::Auto::Jwlwv2::Alienfile> root: C:/Users/cwhitener/_psperl/_perls/perl32w64int-5.30.0.1/data/.cpanm/work/1584319397.3836/Alien-Build-2.14/.tmp/ZjwQGlwvS0/root
    # Alien::Build::Auto::Jwlwv2::Alienfile> stage: C:/Users/cwhitener/_psperl/_perls/perl32w64int-5.30.0.1/data/.cpanm/work/1584319397.3836/Alien-Build-2.14/.tmp/ZjwQGlwvS0/stage
    # Alien::Build::Auto::Jwlwv2::Alienfile> prefix = C/Users/cwhitener/_psperl/_perls/perl32w64int-5.30.0.1/data/.cpanm/work/1584319397.3836/Alien-Build-2.14/.tmp/ZjwQGlwvS0/prefix
    # build threw exception: mkpath failed for C:/Users/cwhitener/_psperl/_perls/perl32w64int-5.30.0.1/data/.cpanm/work/1584319397.3836/Alien-Build-2.14/.tmp/ZjwQGlwvS0/root/destdir_Sb2W/C/Users/cwhitener/_psperl/_perls/perl32w64int-5.30.0.1/data/.cpanm/work/1584319397.3836/Alien-Build-2.14/.tmp: No such file or directory; The filename or extension is too long at C:/Users/cwhitener/_psperl/_perls/perl32w64int-5.30.0.1/data/.cpanm/work/1584319397.3836/Alien-Build-2.14/t/alien_build_plugin_build_autoconf.t line 72.

# Failed test 'out-of-source'
# at t/alien_build_plugin_build_autoconf.t line 107.
t/alien_build_plugin_build_autoconf.t ................ Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/3 subtests
t/alien_build_plugin_build_cmake.t ................... ok
t/alien_build_plugin_build_make.t .................... ok
t/alien_build_plugin_build_msys.t .................... ok
t/alien_build_plugin_build_searchdep.t ............... skipped: test requires Env::ShellWords
t/alien_build_plugin_core_cleaninstall.t ............. ok
t/alien_build_plugin_core_download.t ................. ok
t/alien_build_plugin_core_ffi.t ......................
t/alien_build_plugin_core_ffi.t ...................... 1/? # Failed test 'basic'
# at t/alien_build_plugin_core_ffi.t line 70.
# Caught exception in subtest: mkpath failed for C:/Users/cwhitener/_psperl/_perls/perl32w64int-5.30.0.1/data/.cpanm/work/1584319397.3836/Alien-Build-2.14/.tmp/zoYf2NfKny/root/destdir_99R7/C/Users/cwhitener/_psperl/_perls/perl32w64int-5.30.0.1/data/.cpanm/work/1584319397.3836/Alien-Build-2.14/.tmp: No such file or directory; The filename or extension is too long at C:/Users/cwhitener/_psperl/_perls/perl32w64int-5.30.0.1/data/.cpanm/work/1584319397.3836/Alien-Build-2.14/t/alien_build_plugin_core_ffi.t line 25.

# Failed test 'deprecated'
# at t/alien_build_plugin_core_ffi.t line 135.
# Caught exception in subtest: mkpath failed for C:/Users/cwhitener/_psperl/_perls/perl32w64int-5.30.0.1/data/.cpanm/work/1584319397.3836/Alien-Build-2.14/.tmp/gdEg8bVLfa/root/destdir_256f/C/Users/cwhitener/_psperl/_perls/perl32w64int-5.30.0.1/data/.cpanm/work/1584319397.3836/Alien-Build-2.14/.tmp: No such file or directory; The filename or extension is too long at C:/Users/cwhitener/_psperl/_perls/perl32w64int-5.30.0.1/data/.cpanm/work/1584319397.3836/Alien-Build-2.14/t/alien_build_plugin_core_ffi.t line 91.
t/alien_build_plugin_core_ffi.t ...................... Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/2 subtests
t/alien_build_plugin_core_gather.t ................... 1/?     # Failed test 'bin/foo.exe'
    # at t/alien_build_plugin_core_gather.t line 48.
    # Failed test 'lib/libfoo.a'
    # at t/alien_build_plugin_core_gather.t line 49.

# Failed test 'destdir filter'
# at t/alien_build_plugin_core_gather.t line 52.
t/alien_build_plugin_core_gather.t ................... Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/4 subtests
t/alien_build_plugin_core_legacy.t ................... ok
t/alien_build_plugin_core_override.t ................. ok
t/alien_build_plugin_core_setup.t .................... ok
t/alien_build_plugin_core_tail.t ..................... ok
t/alien_build_plugin_decode_dirlisting.t ............. ok
t/alien_build_plugin_decode_dirlistingftpcopy.t ...... ok
t/alien_build_plugin_decode_html.t ................... ok
t/alien_build_plugin_decode_mojo.t ................... ok
t/alien_build_plugin_download_negotiate.t ............ ok
t/alien_build_plugin_extract_archivetar.t ............ ok
t/alien_build_plugin_extract_archivezip.t ............ ok
t/alien_build_plugin_extract_commandline.t ........... ok
t/alien_build_plugin_extract_commandline__tar_can.t .. ok
t/alien_build_plugin_extract_directory.t ............. ok
t/alien_build_plugin_extract_negotiate.t ............. ok
t/alien_build_plugin_fetch_curlcommand.t ............. ok
t/alien_build_plugin_fetch_httptiny.t ................ ok
t/alien_build_plugin_fetch_local.t ................... ok
t/alien_build_plugin_fetch_localdir.t ................ ok
t/alien_build_plugin_fetch_lwp.t ..................... ok
t/alien_build_plugin_fetch_netftp.t .................. ok
t/alien_build_plugin_fetch_wget.t .................... ok
t/alien_build_plugin_gather_isolatedynamic.t ......... 1/?     # Failed test 'destdir'
    # at t/alien_build_plugin_gather_isolatedynamic.t line 96.
    # Caught exception in subtest: mkpath failed for C:/Users/cwhitener/_psperl/_perls/perl32w64int-5.30.0.1/data/.cpanm/work/1584319397.3836/Alien-Build-2.14/.tmp/cRHR2kP2as/root/destdir_yEvy/C/Users/cwhitener/_psperl/_perls/perl32w64int-5.30.0.1/data/.cpanm/work/1584319397.3836/Alien-Build-2.14/.tmp: No such file or directory; The filename or extension is too long at C:/Users/cwhitener/_psperl/_perls/perl32w64int-5.30.0.1/data/.cpanm/work/1584319397.3836/Alien-Build-2.14/t/alien_build_plugin_gather_isolatedynamic.t line 84.

# Failed test 'basic'
# at t/alien_build_plugin_gather_isolatedynamic.t line 98.
t/alien_build_plugin_gather_isolatedynamic.t ......... Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests
t/alien_build_plugin_meta.t .......................... ok
t/alien_build_plugin_pkgconfig_commandline.t ......... skipped: use PkgConfig::PP on windows
t/alien_build_plugin_pkgconfig_libpkgconf.t .......... skipped: Test requires PkgConfig::LibPkgConf
t/alien_build_plugin_pkgconfig_makestatic.t .......... ok
t/alien_build_plugin_pkgconfig_negotiate.t ........... ok
t/alien_build_plugin_pkgconfig_negotiate__pick.t ..... ok
t/alien_build_plugin_pkgconfig_pp.t .................. ok
t/alien_build_plugin_prefer_badversion.t ............. ok
t/alien_build_plugin_prefer_goodversion.t ............ ok
t/alien_build_plugin_prefer_sortversions.t ........... ok
t/alien_build_plugin_probe_cbuilder.t ................ ok
t/alien_build_plugin_probe_cbuilder__live.t .......... skipped: CI only
t/alien_build_plugin_probe_commandline.t ............. ok
t/alien_build_plugin_probe_vcpkg.t ................... skipped: Test requires Win32::Vcpkg 0.02
t/alien_build_plugin_test_mock.t ..................... ok
t/alien_build_rc.t ................................... ok
t/alien_build_temp.t ................................. ok
t/alien_build_tempdir.t .............................. ok
t/alien_build_util.t ................................. ok
t/alien_build_version_basic.t ........................ ok
t/alien_role.t ....................................... ok
t/alienfile.t ........................................ ok
t/test_alien.t ....................................... ok
t/test_alien_build.t ................................. ok
t/test_alien_cancompile.t ............................ ok
t/test_alien_canplatypus.t ........................... ok
t/test_alien_diag.t .................................. ok
t/test_alien_run.t ................................... ok
t/test_alien_synthetic.t ............................. ok

Test Summary Report
-------------------
t/alien_build_plugin_build_autoconf.t              (Wstat: 256 Tests: 3 Failed: 1)
  Failed test:  3
  Non-zero exit status: 1
t/alien_build_plugin_core_ffi.t                    (Wstat: 512 Tests: 2 Failed: 2)
  Failed tests:  1-2
  Non-zero exit status: 2
t/alien_build_plugin_core_gather.t                 (Wstat: 256 Tests: 4 Failed: 1)
  Failed test:  1
  Non-zero exit status: 1
t/alien_build_plugin_gather_isolatedynamic.t       (Wstat: 256 Tests: 1 Failed: 1)
  Failed test:  1
  Non-zero exit status: 1
Files=78, Tests=391, 70 wallclock secs ( 0.41 usr +  0.06 sys =  0.47 CPU)
Result: FAIL
Failed 4/78 test programs. 5/391 subtests failed.
gmake: *** [Makefile:972: test_dynamic] Error 255
FAIL
! Installing Alien::Build failed. See C:\Users\cwhitener\_psperl\_perls\perl32w64int-5.30.0.1\data\.cpanm\work\1584319397.3836\build.log for details. Retry with --force to force install it.
PS C:\Users\cwhitener\chase\Alien-libuv>
```

**Comment by genio** [comment]
It _does_ however install perfectly when using `cpan` rather than `cpanm`.

**Comment by plicease** [comment]
> It _does_ however install perfectly when using `cpan` rather than `cpanm`.

Yeah it depends entirely on the install directory, which is different for `cpan`.  This should be addressed with #182 and version 2.15.

A deeply nested install directory could cause other problems, but at least this one is easy to work around unless the end user has a deeply nested temp directory (which is tbh less likely).
